File: POO/temas/8/py/8.py
class Frigo:
    def __init__(self,color):
        self.__color = color
        self.__puerta = "cerrada"
        self.__temperatura = 20

    def abre(self):
        self.__puerta = "abierta"

    def cierra(self):
        self.__puerta = "cerrada"

    def fija_temperatura(self,nueva_temperatura):
        self.__temperatura = nueva_temperatura

    def muestra_estado(self):
        print( f'Color: {self.__color} | Puerta {self.__puerta} | {self.__temperatura}ºC' )


mi_congelador = Frigo("rojo")
mi_congelador.muestra_estado()

otro =  Frigo("blanco")
otro.muestra_estado()

File: POO/temas/8/rb/8.rb
class Frigo
    def initialize
        @color = "blanco"
        @puerta = "cerrada"
        @temperatura = 20
    end

    def abre
        @puerta = "abierta"
    end

    def cierra
        @puerta = "cerrada"
    end

    def fija_temperatura(nueva_temperatura)
        @temperatura = nueva_temperatura
    end

    def muestra_estado
        puts "Color: #{@color} | Puerta #{@puerta} | #{@temperatura}ºC"
    end
end


mi_frigo = Frigo.new
mi_frigo.muestra_estado
mi_frigo.fija_temperatura(-10)
mi_frigo.muestra_estado
